src/secure-proxy-handler.ts

```diff
@@ -232,14 +224,12 @@ export class SecureProxyHandler implements ProxyHandler<SecureProxyTarget> {
     setPrototypeOf(shadowTarget: SecureShadowTarget, prototype: SecureValue): boolean {
         this.initialize(shadowTarget);
         // this operation can only affect the env object graph
-        setPrototypeOf(shadowTarget, prototype);
-        return true;
+        return setPrototypeOf(shadowTarget, prototype);
```

Comment:
from reflect? do we eve user setPrototypeOf from object? maybe we can rename these utilities to `ReflectSetPrototypeOf` or something like that so it is very clear where they are coming from.

Reply:
Good idea.